common/src/main/java/tds/common/configuration/JacksonObjectMapperConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package tds.common.configuration;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JacksonObjectMapperConfiguration {
+    @Bean
+    public Jackson2ObjectMapperBuilderCustomizer customizeObjectMapper() {
+        return builder -> {
+            builder.serializationInclusion(JsonInclude.Include.NON_NULL);
+            builder.modules(new GuavaModule(), new JodaModule());
+        };
+    }
```

Comment:
Below is an example JSON for exam.  The `test` field is a Guava optional only included for testing purposes. 

```{json}
{
  "id": "537fe516-ad8e-4859-84ca-e478d86ad16c",
  "sessionId": "85ab1cc2-368a-4ca8-89f0-824163f09091",
  "browserId": "16c4b46a-a0a0-4783-9dab-1bc61d6ab555",
  "assessmentId": "assessmentId",
  "studentId": 1,
  "attempts": 0,
  "maxItems": 6,
  "status":{
  "status": "pending",
  "stage": "IN_USE"
  },
  "statusChangeDate": 1482858675374,
  "clientName": "clientName",
  "subject": "ELA",
  "createdAt": 1482858675651,
  "loginSSID": "ADV001",
  "studentName": "Darth",
  "dateJoined": 1482858675368,
  "assessmentWindowId": "ANNUAL",
  "assessmentAlgorithm": "fixedForm",
  "assessmentKey": "(SBAC_PT)SBAC-IRP-CAT-ELA-3-Summer-2015-2016",
  "environment": "Development",
  "languageCode": "ENU",
  "segmented": false,
  "abnormalStarts": 5,
  "waitingForSegmentApproval": false,
  "currentSegmentPosition": 1,
  "test": "TESTING"
}
```